sway/commands/allow_tearing.c

```diff
@@ -0,0 +1,25 @@
+#include <sway/commands.h>
+#include "sway/config.h"
+#include "sway/tree/view.h"
+#include "util.h"
+
+struct cmd_results *cmd_allow_tearing(int argc, char **argv) {
+  struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "allow_tearing", EXPECTED_AT_LEAST, 1))) {
+		return error;
+	}
+
+  struct sway_container *container = config->handler_context.container;
+  if (!container || !container->view) {
+    return cmd_results_new(CMD_INVALID,
+				"Tearing can only be allowed on views");
+  }
+
+  bool wants_tearing = parse_boolean(argv[0], true);
```

Comment:
Should we have a way to reset this back to the default?

Reply:
I my mind, if you want the default behavior, you just remove the window rule. I can't think of a reason why you would `explicitly` set it to the default behavior?
 
 But if you do think we need it, maybe we could add an `auto` option?

Reply:
I'm thinking of users experimenting with the new commands, toggling to `on` and then trying to go back to the defaults. But yeah, I agree that it's easy enough to just reload the config file, and a way to reset to the default can be introduced later if it turns out it's desirable.